test/units/TEST-74-AUX-UTILS.mount.sh

```diff
@@ -43,6 +43,9 @@ mount "$LOOP" "$WORK_DIR/mnt"
 touch "$WORK_DIR/mnt/foo.bar"
 umount "$LOOP"
 (! mountpoint "$WORK_DIR/mnt")
+# Wait for the mount unit unloaded. Otherwise, creating transient unit below may be failed.
```

Comment:
"Wait for the mount unit to be unloaded. Otherwise, creation of the transient unit below may fail."